File: backend/src/main/java/com/luan/meeting/room/meeting_room/model/Room.java
package com.luan.meeting.room.meeting_room.model;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "meeting_room")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Room {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "name", nullable = false)
    private String name;

    @Column(name = "date", nullable = false)
    private String date;

    @Column(name = "startHour", nullable = false)
    private String startHour;

    @Column(name = "endHour", nullable = false)
    private String endHour;

    @Override
    public String toString() {
        return String
                .format("Name: %s | Date: %s | Start Hour: %s | End Hour: %s", name, date, startHour, endHour);
    }
}

File: frontend/src/app/room.ts
export class Room {
  id?: number;
  name?: string;
  date?: string;
  startHour?: string;
  endHour?: string;
  active?: boolean;

  constructor(
    id: number = 0,
    name: string = '',
    date: string = '',
    startHour: string = '',
    endHour: string = '',
    active: boolean = true
  ) {
    this.id = id;
    this.name = name;
    this.date = date;
    this.startHour = startHour;
    this.endHour = endHour;
    this.active = active;
  }
}
